src/constants/rules.js

```diff
@@ -0,0 +1,17 @@
+/** 로또 게임의 룰과 관련된 상수들입니다. */
+export const LOTTO = Object.freeze({
```

Comment:
```suggestion
export const LOTTO_RULES = Object.freeze({
```
`LOTTO`라는 변수명으로 보고 상수화 되어있는 객체라고 파악하기는 어려울 것 같아서 `LOTTO_RULES`와 같이 수정해보는건 어떠신가요?!